torch/csrc/generic/StorageMethods.cpp

```diff
@@ -179,6 +179,31 @@ static PyObject * THPStorage_(fromBuffer)(PyObject *_unused, PyObject *args, PyO
 }
 #endif
 
+static PyObject * THPStorage_(fromFile)(PyObject *_unused, PyObject *args, PyObject *keywds)
+{
+  HANDLE_TH_ERRORS
+  const char *filename;
+  Py_ssize_t size = 0;
+  int shared = 0, shared_mem = 0;
+  static char *kwlist[] = {"filename", "shared", "size", "shared_mem", NULL};
+  if (!PyArg_ParseTupleAndKeywords(args, keywds, "s|pnp", kwlist,
+              &filename, &shared, &size, &shared_mem)) {
+    return NULL;
+  }
+#if defined(__APPLE__)
+  shared_mem = 0;
```

Comment:
Why is it useless? Isn't the filed unlinked when the storage is destructed? In theory you could open multiple views